turbo.json

```diff
@@ -7,8 +7,7 @@
         "NODE_ENV",
         "DATABASE_URL",
         "DIRECT_URL",
-        "NEXTAUTH_SECRET",
-        "NEXTAUTH_URL",
+        "BETTER_AUTH_SECRET",
```

Comment:
leave in NEXTAUTH_SECRET so it doesnt break for self hosters on vercel

Reply:
to keep next auth secret support we only need to keep the turbo repo mention and the process.env mention here: `secret: env.AUTH_SECRET || process.env.NEXTAUTH_SECRET,`. you can remove the rest of the mentions